helmfile.d/stacks/calico.yaml.gotmpl

```diff
@@ -2,6 +2,7 @@
 templates:
   calico:
     condition: ck8sAnyCluster.enabled
+    installed: {{ .Values | get "networkPlugin.type" | eq "calico" }}
```

Comment:
```suggestion
```
No `installed` in templates that are expected to be inherited with other `installed`.
(Based on the discussion we had some release ago, as helmfile will not merge `false` over `true`.)